hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractEvictableRecordStore.java

```diff
@@ -287,6 +280,8 @@ protected void markRecordStoreExpirable(long ttl) {
 
     abstract Object evictInternal(Data key, boolean backup);
 
+    abstract Object evictInternal(Data key, Record removedRecord, boolean backup);
```

Comment:
What does this mean? Please add javadoc. Since there is a complex architecture for the Map, documentation is required so that other engineers can work easily with map as well.